google-cloud-tools-plugin/src/com/google/gct/idea/debugger/UpdateIdeWithBreakpointAction.java

```diff
@@ -0,0 +1,73 @@
+package com.google.gct.idea.debugger;
+
+import com.google.api.client.repackaged.com.google.common.base.Strings;
+import com.google.api.services.clouddebugger.model.Breakpoint;
+
+import com.intellij.debugger.ui.breakpoints.BreakpointManager;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.xdebugger.breakpoints.XBreakpoint;
+import com.intellij.xdebugger.breakpoints.XBreakpointManager;
+import com.intellij.xdebugger.breakpoints.XLineBreakpoint;
+
+import java.util.Map;
+
+class UpdateIdeWithBreakpointAction implements Runnable {
```

Comment:
Yeah, I was contemplating on it as well, but calling it Runnable would have been redundant, Action seemed a bit better. Maybe `UpdateIdeWithBreakpointTask`?